app/models/reservation.rb

```diff
@@ -103,6 +103,25 @@ class Reservation < ActiveRecord::Base
       .select('reservations.*, equipment_models.category_id as category_id')
   }
 
+  # for status modifying jobs
+  def self.deletable_missed
```

Comment:
Should we make these query objects instead? To me that feels better than further expanding our already bloated Reservation model. We can still tie them to named scopes as we've done with the other query objects.

On the other hand, these aren't strictly speaking scopes since we won't use them anywhere else or chain them with other scopes. In that case, I feel like we're definitely better off just creating separate objects (or alternatively just including this logic inside the job instead, since it's where the actual responsibility lies) - I'm just not sure that `Reservation` needs to know where deletable missed reservations are, for example.

I'm not 100% convinced of what I'm suggesting above, so let me know what you think and we'll figure it out together :stuck_out_tongue:.